### DISCUSSION TITLE
Slower camera-orbit?

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4299] [Q&A] [answered]
Hi guys,
I have a function that resets the view of the model and it goes like this:

`resetView.addEventListener('click', function () {
    modelViewer.setAttribute('camera-orbit', "35.29deg 67.94deg auto");
    modelViewer.setAttribute('field-of-view', "30deg");
    modelViewer.setAttribute('camera-target', "auto auto auto");
    UpdateFrameing();
});
`

Is there a way to slow down the transition? It happens kind of fast and mr boss man asked if I can slow it down but I can't seem to figure it out.

**Comment by AntonioMartinezC** [comment]
I am working on a similar issue atm. I saw an issue on that topic but apparently I didn't safe it. 

Apparently the camera moves from new to old position in 500ms and with fast acceleration in the beginning if I recall it correctly. They talked about a way to basically compile the lib yourself with a new value. 

If I get anywhere with my efforts (or find that issue again) I will come back to this

**Reply by elalish** [reply, marked as answer]
I believe you're both looking for this: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-interpolationDecay

**Reply by danilokrulj** [reply]
As always, you are more than helpful, thank you!
